frontend/src/metabase/dashboard/selectors.js

```diff
@@ -178,15 +178,12 @@ export const getParameters = createSelector(
   },
 );
 
-export const makeGetParameterMappingOptions = () => {
-  const getParameterMappingOptions = createSelector(
-    [getMetadata, getEditingParameter, getCard, getDashCard],
-    (metadata, parameter, card, dashcard) => {
-      return _getParameterMappingOptions(metadata, parameter, card, dashcard);
-    },
-  );
-  return getParameterMappingOptions;
-};
+export const getParameterMappingOptions = createSelector(
```

Comment:
This is just refactoring. I have no clue why we need to have the function factory when we don't ever pass any argument to it.

Reply:
I think the idea was to create a "reselect" selector instance per component to avoid issues with memoization. But the implementation was wrong and it didn't work.